Dockerfile.release

```diff
@@ -0,0 +1,31 @@
+FROM phusion/baseimage
+
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+ENV LC_ALL en_US.UTF-8
+
+# required packages
+RUN apt-get update && apt-get upgrade -y && apt-get install -y \
+    bash \
+    bash-completion \
+    curl \
+    dnsutils \
+    vim && \
+    apt-get clean
+
+ENV FENNEC_UDP_BIND_IP=0.0.0.0 FENNEC_UDP_PORT=3478 MIX_ENV=prod \
+    FENNEC_TCP_BIND_IP=0.0.0.0 FENNEC_TCP_PORT=3479 MIX_ENV=prod \
+    REPLACE_OS_VARS=true SHELL=/bin/bash
```

Comment:
Right. I think makes distillery replace some vars in sys.config and vm.args before starting the release.